src/resource_dasm.cc

```diff
@@ -114,7 +115,16 @@ class ResourceExporter {
           case '%':
             result += '%';
             break;
-            
+          
+          case 'a':
+            result += res_flags & FLAG_COMPRESSED ? 'c' : '-';

```

Comment:
Currently, this will probably never be set unless a resource failed in decompression. We could later add a case for FLAG_DECOMPRESSED to show which output files came from resources that were originally compressed.